src/instruction.rs

```diff
@@ -116,8 +574,28 @@ impl fmt::Debug for Operation {
             Operation::NOP => write!(f, "NOP"),
             Operation::PUSH(val) => write!(f, "PUSH\t{:#x?}", val),
             Operation::JMP(val) => write!(f, "JMP\t{:#x?}", val),
-            Operation::MVI(lhs, rhs) => write!(f, "MVI\t{:#x?},{:#x?}", lhs, rhs),
+            Operation::MVI(lhs, rhs) => write!(f, "MVI\t{:#x?}, {:#x?}", lhs, rhs),
```

Comment:
I did not have a space here as I don't believe most asm commands with two operands contain a space after the comma. I might be wrong though. Either way it's not a big deal - just need to be consistent throughout so let's just stick with the space formatting here.